internal/venice-consumer/build.gradle

```diff
@@ -8,7 +6,7 @@ dependencies {
   implementation libraries.log4j2api
 
   testImplementation project (':clients:venice-thin-client')
-  testImplementation project (':internal:venice-test-common').sourceSets.integrationTest.output
+  testImplementation project(path: ':internal:venice-test-common', configuration: 'integrationTestUtils')
```

Comment:
There is not much in this module that needs integration test dependency. Howerver, since your change doesn't relate to this, it is okay to make this change.

We should eventually try to remove this dependency, though